zellij-server/src/pty.rs

```diff
@@ -468,7 +471,8 @@ impl Pty {
         layout: PaneLayout,
         default_shell: Option<TerminalAction>,
         client_id: ClientId,
-    ) {
+    ) -> Result<()> {
+        let err_context = || format!("fail to spawn terminals layout for client {client_id}");
```

Comment:
Sorry, this is picky, but I think we should be consistent with the error phrasing

```suggestion
        let err_context = || format!("failed to spawn terminals for layout for client {client_id}");
```